### ISSUE TITLE
Irradiated Comb has no use

### ISSUE THREAD

**Issue by Pyromancer56** [JasonMcRay/InfiTech-1.7#696]
#### Issue Description:
Irradiated Combs from Vindictive, Vengeful, and Avenging bees currently do not centrifuge into anything. I assume this is because it would typically give Iridium Ore.

____

**Comment by Xavion3** [comment]
Actually technically this is not a bug, the relevant code is [here](https://github.com/ForestryMC/ForestryMC/blob/mc-1.7.10/src/main/java/forestry/plugins/PluginApiculture.java#L674-L676). We could add a use yes so I won't close this but them not giving anything is a vanilla forestry thing.

**Comment by AiwendilCiunio** [comment]
I think this is intended. It's irradiated, duh. Maybe it could give cancer though :D.

**Comment by Timeslice42** [comment]
Looks like a bad copy/paste to me. There should at least be a wax and/or honey given.

**Comment by JasonMcRay** [comment]
I took a look at it so:

1) Yes it used to be cetrifuged to Irridium ore. But we already have Irridium Bee for the multiplication purposes, so there is no reason to have the Irradiated Combs give Iridium
2) I tried to make it produce at least honey, but since GT is loading recipes directly from forestry I cannot do anything with MT. No matter what I do, GT Centrifuge will still keep the native forestry recipe
3) Yes I have looked into recipes.cfg but there is nothing in the centrifuge section for any of the combs (except those we are adding)

So I am left with leaving it as it is, making the Radioactive Line only useful for their traits as Longest life span, huge pollination area and for breeding higher to Uranium bees.

I could just remove/blacklist them but I would need to make changes to the breeding chart, and I am too lazy to do that.
